cell_based/test/tutorial/TestRunningMeshBasedSimulationsTutorial.hpp

```diff
@@ -196,7 +196,7 @@ class TestRunningMeshBasedSimulationsTutorial : public AbstractCellBasedTestSuit
      * java executable.
      *
      * For further details on visualization, see
-     * [ChasteGuides/RunningCellBasedVisualization](https://chaste.cs.ox.ac.uk/trac/wiki/ChasteGuides/RunningCellBasedVisualization).
+     * [Chaste Visualisation Guides](https://chaste.github.io/docs/user-guides/visualisation-guides/).
```

Comment:
This might want to be relative so that an archived version of Chaste 2024.whatever tutorials points to the relevant visualisation guide in the same archive too? But I always get confused about how to do that

Reply:
I've fixed the relative link in 61e1c33